src/DynamoCoreWpf/Utilities/Md2Html.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using Dynamo.Utilities;
+using DynamoUtilities.Properties;
+
+namespace Dynamo.Wpf.Utilities
+{
+    /// <summary>
+    /// Utilities for converting Markdown to html and for sanitizing html
+    /// The Md2Html command line tool is used for doing the actual conversion/santizing
+    /// (This tool is delivered as part of Dynamo)
+    /// This class is not thread safe so please instantiate this class in the same thread that
+    /// you intend to use.
+    /// But multiple instances of the class is supported.
+    /// </summary>
+
+    internal class Md2Html : CLIWrapper
+    {
+        private string relativePath = Path.Combine(@"Md2Html", @"Md2Html.exe");
+        /// <summary>
+        /// Constructor
+        /// Start the CLI tool and keep it around
+        /// </summary>
+        internal Md2Html()
+        {
+            StartProcess(relativePath,string.Empty);
+        }
+
+        /// <summary>
+        /// Kill the CLI tool, if still running
+        /// </summary>
+        /// <param name="disposing"></param>
+        protected virtual void Dispose(bool disposing)
+        {
+            KillProcess();
+        }
+
+        /// <summary>
+        /// Kill the CLI tool, if still running
+        /// </summary>
+        public override void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Converts a markdown string into Html.
+        /// </summary>
+        /// <param name="mdString"></param>
+        /// <param name="mdPath"></param>
+        /// <returns>Returns converted markdown as html</returns>
+        internal string ParseMd2Html(string mdString, string mdPath)
```

Comment:
why is this class being moved to DynamoCoreWPF? MD2HTML conversion could conceivably be useful without WPF - correct?

Reply:
It could but it is not at the moment it is only used from WPF projects.
The purpose of this change was to remove resources from DynamoUtilities.

Reply:
I could move it to DynamoCore just as easily.

Reply:
nah, this is fine, but could you explain why moving resources out of DynamoUtiltiies is useful?

Reply:
It is more about cleanup than anything else. My thinking is to minimize the use of resources in core dlls (less dlls, less time building etc)

As part of the System.Drawing task there is some cleanup of resources (ex removing System.Drawing from resources if not needed). This PR is about removing entire resource dlls if not needed.

Anyway this PR is still in draft because I wanted to check with the team if it seems like a good approach.